extensions/wikia/SyntaxHighlight/tests/SyntaxHighlightTest.php

```diff
@@ -17,6 +17,8 @@ protected function setUp() {
 		$this->setupFile = __DIR__ . '/../SyntaxHighlight.setup.php';
 		parent::setUp();
 
+		Hooks::register( 'ParserFirstCallInit', 'SyntaxHighlightHooks::onParserFirstCallInit' );
+
```

Comment:
?

Reply:
We've unset the hooks in other test, so it's better to register it here